src/main/java/Deadline.java

```diff
@@ -0,0 +1,20 @@
+public class Deadline extends Task {
+    protected final String sym = "D";
+    protected String day;
+
+    Deadline (String description, String day) {
+        super(description);
+        this.day = day;
+    }
+
+    @Override
+    String getSym() {
```

Comment:
I would opt for a more intuitive name rather than sym.

Reply:
> I would opt for a more intuitive name rather than sym.

Maybe you can change it to a more descriptive name such as symbol?